[DOCFIX] Improve scalability doc (#14402)
Alluxio/alluxio: docs/en/operation/Scalability-Tuning.md

For example, in a deployment with 2 users, 50 Presto worker nodes (with 200 task concurrency), and
50 Alluxio nodes, the estimations would come out to the following

> * 50 (workers) + 2 (users) x (11 (block pool size) + 10 (file pool size)) x 50 (services) = 2150

Author: This calculation was wrong. But not sure if I'm now right.

Reviewer: I believe the entire `Number of Concurrent Clients` section is a stretch.
I don't think we can provide meaningful numbers here that covers all possible client applications. (Concurrency setting could be executed at a single FS or multiple FSes depending on the application.)

I vote for dropping the section. @yuzhu WDYT?

Author: +1 

I don't think this number is ever validated. Now that we have better visibility (and we can add more), we should perform some validation and add a section later which we guarantee the correctness.

Reviewer 2: yeah. i agree

It is also important to monitor key timer metrics, as an abnormally high response rate would indicate the master is under stress.
* Master.JournalFlushTimer 

> If the journal can’t keep up with the flush, master request per second might be too high. Consider using a more powerful master node.

Reviewer: ```suggestion
If the journal can’t keep up with the flush, master might report request count per second that is higher than initiated by clients. Consider using a more powerful master node.
```

* Master.ListStatus Timer

> Any of the RPC timer statistics would help here. If the latency is abnormally high, master is under a lot of load. 

Reviewer: ```suggestion
Any of the RPC timer statistics would help here. If the latency is abnormally high, master might be under a lot of load. 
```

> It might be time for a more powerful master node.

Reviewer: ```suggestion
Consider using a more powerful master node.
```
